File: Selenium/Activities/Activity19.java
package examples;

import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;

public class Activity19 {

	public static void main(String[] args) {

			WebDriver driver = new FirefoxDriver();
			
			driver.get("https://training-support.net/webelements/alerts");
			System.out.println("The page title is : "+ driver.getTitle());
			
			WebElement btnConfirm =  driver.findElement(By.cssSelector("button#confirmation"));
			WebElement result = driver.findElement(By.cssSelector("p#result"));
			
			btnConfirm.click();
			
			Alert alert = driver.switchTo().alert();
			System.out.println("The text on the simple alert is : "+alert.getText());
			alert.accept();
			System.out.println("Result message is : " + result.getText());
			
			btnConfirm.click();
			
			driver.switchTo().alert();
			System.out.println("The text on the simple alert is : "+alert.getText());
			alert.dismiss();
			System.out.println("Result message is : " + result.getText());
			
			driver.quit();

	}

}

File: Selenium/Activities/Activity19.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.select import Select

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/alerts")
    print(f"The page title is : {driver.title}")

    btnConfirm =  driver.find_element(By.CSS_SELECTOR,"button#confirmation")
    result = driver.find_element(By.CSS_SELECTOR,"p#result")

    btnConfirm.click()

    alert = driver.switch_to.alert
    print(f"The text on the simple alert is : {alert.text}")
    alert.accept()
    print(f"Result message is : {result.text}")
			
    btnConfirm.click()
    driver.switch_to.alert
    print(f"The text on the simple alert is : {alert.text}")
    alert.dismiss()

    print(f"Result message is : {result.text}")
    driver.quit()